.github/workflows/build.yml

```diff
@@ -12,29 +12,41 @@ on:
 jobs:
     tests:
         runs-on: ubuntu-latest
-        name: "PHP ${{ matrix.php }}${{ matrix.symfony != '' && format(', Symfony {0}', matrix.symfony) || '' }}${{ matrix.currency-bundle != '' && format(', CurrencyBundle {0}', matrix.currency-bundle) || '' }}"
+        name: "PHP ${{ matrix.php }}${{ matrix.symfony != '' && format(', Symfony {0}', matrix.symfony) || '' }}${{ matrix.currency-bundle != '' && format(', CurrencyBundle {0}', matrix.currency-bundle) || '' }}${{ matrix.postgres != '' && format(', PostgresSQL {0}', matrix.postgres) || '' }}"
         strategy:
             fail-fast: false
             matrix:
                 php: [7.4, 8.0]
+                postgres: [""]
                 symfony: [^4.4, ^5.3]
 
                 include:
                     -
                         php: 7.4
                         symfony: ""
                         currency-bundle: "^1.7"
+                    -
+                        php: 8.0
+                        symfony: "^5.3"
+                        postgres: "13.3"
```

Comment:
I'm wondering if we shouldn't test v14 as well. Also, if we are testing with postgres, shouldn't we trigger build with MySQL? At the end, we are testing SqlLite only right now

Reply:
Yes I can add the v14 for Postgres, but I think the MySQL build can be added in another PR.

Reply:
done for PostgreSQL v14